api/envoy/api/v2/core/base.proto

```diff
@@ -76,6 +77,39 @@ message Locality {
   string sub_zone = 3;
 }
 
+// BuildVersion combines SemVer version of extension with free-form build information
+// (i.e. 'alpha', 'private-build') as a set of strings.
+message BuildVersion {
+  // SemVer version of extension
+  type.SemanticVersion version = 1;
+
+  // Free-form build information i.e. 'alpha'
+  repeated string build_info = 2;
+}
+
+// Version and identification for an Envoy extension.
+message Extension {
+  // This is the name of the Envoy filter as specified in the Envoy
+  // configuration, e.g. envoy.router, com.acme.widget.
+  string name = 1;
+
+  // Category of the extension
+  string category = 2;
+
+  // The version is a property of the extension and maintained independently
+  // of other extensions and the Envoy API.
+  BuildVersion version = 3;
+
+  // Indicates that the extension is present but was disabled during runtime.
```

Comment:
I think runtime has a very specific meaning in Envoy (the Runtime system). Extensions can be disabled via configuration. I think it should just say "but was disabled via dynamic configuration".

Reply:
done